SlevomatCodingStandard/Sniffs/Functions/NamedArgumentSpacingSniff.php

```diff
@@ -0,0 +1,79 @@
+<?php declare(strict_types = 1);
+
+namespace SlevomatCodingStandard\Sniffs\Functions;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+use SlevomatCodingStandard\Helpers\TokenHelper;
+use function sprintf;
+use const T_COLON;
+use const T_PARAM_NAME;
+use const T_WHITESPACE;
+
+class NamedArgumentSpacingSniff implements Sniff
+{
+
+	public const CODE_WHITESPACE_BEFORE_COLOR = 'WhitespaceBeforeColon';
```

Comment:
Noticed there is a typo in constant name - COLOR instead of COLON.

Reply:
Thanks, fixed.

Reply:
👍